distribution/release-seata.xml

```diff
@@ -35,7 +35,9 @@
 
         <fileSet>
             <includes>
-                <include>bin/*</include>
+                <include>bin/seata-server.bat</include>
```

Comment:
why change?

Reply:
`bin/*` contains both seata-server.sh and seata-namingserver.sh, one should be included in release-seata.xml while the other should be included in release-namingserver.xml.

Reply:
> `bin/*` contains both seata-server.sh and seata-namingserver.sh, one should be included in release-seata.xml while the other should be included in release-namingserver.xml.

ok